components/BackButton/index.js

```diff
@@ -21,7 +23,7 @@ const Back = styled.span`
     border-radius: 30%;
   }
 `;
-const StyledLink = styled(Link)`
-  transition: 0.3s ease-in-out;
 
+const BackButtonContainer = styled.div`
```

Comment:
This should be an interactive element. A Button or a Link, since we are linking to another page... a Link maybe? 
We should take care of accessability and semantic HTML. A div has no semantic meaning and sreanreader will not be able to navigate properly.

Reply:
I replaced a button with a link. To make it more accessible for screen readers i added:  aria-hidden="true" - to hide the icon from the screen readers and added a text, which is hidden but the SR can read it.